addons/l10n_ch/models/res_bank.py

```diff
@@ -83,19 +128,45 @@ def _onchange_set_l10n_ch_postal(self):
                 self.l10n_ch_postal = self.acc_number.split(" ")[0]
             else:
                 self.l10n_ch_postal = self.acc_number
-                if self.partner_id:
-                    self.acc_number = self.acc_number + '  ' + self.partner_id.name
+                # In case of ISR issuer, this number is not
+                # unique and we fill acc_number with partner
+                # name to give proper information to the user
+                if self.partner_id and self.acc_number[:2] in ["01", "03"]:
```

Comment:
use ISR_SUBSCRIPTION_CODE or don't use it at all :point_up:

Reply:
'01' and '03' doesn't refer to ISR_SUBSCRIPTION_CODE.keys() ?

Reply:
It is probably encoded in the same way but the fields l10n_ch_postal and l10n_ch_subscription... are different things.